src/js/hca/components/insurance-information/MedicareMedicaidSection.jsx

```diff
@@ -30,13 +30,10 @@ class MedicareMedicaidSection extends React.Component {
       dateRequired = true;
 
       medicarePartADateInput = (
-        <DateInput required={dateRequired}
-            errorMessage="Please enter a vaild date."
+        <ErrorableCurrentOrPastDate required={dateRequired}
```

Comment:
Looks like we could remove the `dateRequired` part of this, since it's always true.